libraries/src/Application/WebApplication.php

```diff
@@ -413,7 +413,7 @@ protected function loadSystemUris($requestUri = null)
 			else
 			{
 				// Normalise slashes.
-				$mediaURI = trim($mediaURI, '/\\');
+				$mediaURI = trim($mediaURI  ?? '', '/\\');
```

Comment:
```suggestion
				$mediaURI = trim($mediaURI ?? '', '/\\');
```
Fix PHPCS for drone.